package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "dependencies": {
+    "save-dev": "^0.0.1-security"
```

Comment:
What is it for here?

Reply:
I don`t know what is it and how is appeared here. May I delete it?